### PR TITLE
Footer refresh - Fix typo, add IRC channel, raplace slack with mattermost

### LINKED ISSUE #426: Mattermost site missing from 'Community' menu
The website lists a number of applications and sites in the 'Community' category:

- Reddit
- Stack Exchange
- BitcoinTalk Thread
- Slack Chat
- Telegram Chat

It is missing Mattermost (mattermost.getmonero.org.)

### PR DESCRIPTION
**Pull request by erciccione**
s/channel/channels
add IRC channel #monero-translation
replace Slack with MatterMost

closes #426

### CODE REVIEW

**Comment by QuickBASIC** [comment]
Why not keep both Slack and Mattermost listed? There are still plenty of people using Slack, so I don't think it makes sense to remove it yet.

**Comment by anonimal** [comment]
>There are still plenty of people using Slack, so I don't think it makes sense to remove it yet.

I think this brings up a good question of how to effectively migrate away from Slack to mattermost. Referencing https://github.com/monero-project/meta/issues/81.

**Comment by erciccione** [comment]
I thought MatterMost was meant to replace slack, we shouldn't encourage the use of a proprietary software when we have a self-hosted and open source alternative.

**Comment by QuickBASIC** [comment]
> I thought MatterMost was meant to replace slack, we shouldn't encourage the use of a proprietary software when we have a self-hosted and open source alternative.

Agreed. I withdraw my suggestion. They're nearly functionally identical so it doesn't make sense to direct people to both.

**Comment by erciccione** [comment]
>Can we change the PR title to something more descriptive though?

Edited

File: _includes/footer.html [hunk 1]
```diff
@@ -11,13 +11,14 @@
                         </ul>
                     </div>
                     <div class="col-sm-3 col-xs-6">
-                        <h3>IRC Channel</h3>
+                        <h3>IRC Channels</h3>
```
Comment (mattcode55) [inline _includes/footer.html]: Not sure if this section should be kept because it just duplicates information from the hangouts page.

Comment (erciccione) [reply _includes/footer.html]: What about changing to "IRC Main Channels"? o refer to the complete list in Hangouts? I think a short list of the main channels is useful. @rehrar thoughts on this?

**Comment by rehrar** [comment]
How do I not get the notifications for this thing? My spring cleaning update has the way I think it should be.

**Pull request merged**